client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditForm.jsx

```diff
@@ -660,6 +697,21 @@ const SubmissionEditForm = (props) => {
     />
   );
 
+  const renderErrorMessages = () => (
```

Comment:
we want the error message to be persistent inside the page, so that when user navigates through the submission edit page, they can always go back to some specific location inside the page to see which questions got problems.

if we use toast, it will disappear after some time, and let's say user has too many errors in various number of questions, they need to navigate through all the questions, look carefully on which one got error, and then attempt to fix it. this might be cumbersome for users, and hence we utilised the error message as such..